pdft/pdft.py

```diff
@@ -79,11 +77,13 @@ def basis_to_grid(mol, mat, blocks=True):
     x, y, z= np.array(fullx), np.array(fully), np.array(fullz)
     full_mat = np.array(full_mat)
     full_w = np.array(fullw)
-        
-    if blocks is True:
-        return frag_mat, [frag_x, frag_y, frag_z, frag_w]
-    if blocks is False: 
-        return full_mat, [x,y,z,full_w]
+
+    # A bug here.
+    # if blocks is True:
+    #     return frag_mat, [frag_x, frag_y, frag_z, frag_w]
+    # if blocks is False:
+    #     return full_mat, [x,y,z,full_w]
+    return full_mat, [x,y,z,full_w]
```

Comment:
We may require to get grid data on blocks. Not the default 
```suggestion

if blocks is True:
    return frag_mat, [frag_x, frag_y, frag_z, frag_w]
if blocks is False:
    return full_mat, [x,y,z,full_w]
```